tlm-stencil/src/components/mw-checkbox/mw-checkbox.tsx

```diff
@@ -0,0 +1,37 @@
+import {Component, getAssetPath, h, Prop} from '@stencil/core';
+
+@Component({
+  tag: 'mw-checkbox',
+  styleUrl: 'mw-checkbox.css',
+  assetsDirs: ['../assets'],
+  shadow: true,
+})
+export class MwCheckbox {
+  @Prop() value: string;
+  @Prop() name: string;
+  @Prop() disabled?: boolean;
+  @Prop({mutable:true}) checked?: boolean;
+
+  handleCheck = () => {
+    if (!this.disabled) {
+      this.checked = !this.checked
+    }
+  }
+
+  render() {
+    return (
+      <div class="mw-checkbox-container" onClick={this.handleCheck}>
+        <input type="checkbox" checked={this.checked} value={this.value} name={this.name}/>
+        <span class={`mw-checkbox ${this.checked ? "selected" : "unselected"} ${this.disabled ? "disabled" : "enabled"}`}>
+          <img
+            class={`mw-checkmark ${this.checked ? "selected" : "unselected"}`}
+            src={getAssetPath("../assets/mw-checkbox_icon.svg")}
```

Comment:
I would suggest to move the assets folder one step up, next to components instead of inside of it